.github/workflows/docker.yml

```
@@ -286,6 +293,13 @@ jobs:
           GITEA_DEPLOY_IMAGE_TAG: ':${{ needs.get_version.outputs.gitea_sha_short }}'
           browser: ${{ matrix.browser }}
           group: ${{ matrix.browser }}
+          BUILD_ID: ${{ steps.uuid.outputs.value }}
```

```suggestion
          BUILD_ID: ${{ github.run_id }}-${{ github.run_attempt }}
```

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context